WcaOnRails/app/views/incidents/_nav.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container-fluid">
+  <div class="row">
+    <div id="wrc-nav">
+      <div class="list-group list-group-root">
+        <%= link_to "Incidents log", incidents_path, class: "list-group-item" %>
+        <%= link_to "Some navbar test", incidents_path, class: "list-group-item" %>
+        <%= link_to "Maybe useless", incidents_path, class: "list-group-item" %>
+      </div>
+    </div>
+    <div id="wrc-data">
+      <%= yield %>
+    </div>
+  </div>
+</div>
+
+<script>
+  (function() {
+    var $nav = $('#wrc-nav > .list-group');
+    $nav.affix({
+      offset: {
+        top: function() {
+          var parentTop = $nav.parent().offset().top;
+          var affixTopSpacing = 20; // From .bs-affix in wca.scss
+          return parentTop - affixTopSpacing;
+        },
+        bottom: function () {
+          return (this.bottom = $('.footer').outerHeight(true));
+        },
+      },
+    });
+  })();
```

Comment:
This is the copy of [this code](https://github.com/thewca/worldcubeassociation.org/blob/7fe29ddc36aa4b959984295d2435f17b3b1c8683/WcaOnRails/app/views/competitions/_nav.html.erb#L200-L214) and the only difference is the `$nav` selector, so this should certainly be a generic thing (e.g. applied to all the items with a `data-affixed-nav` attribute).
Also the comment seems to be outdated ^^

Reply:
I agree with @jonatanklosko about may making a nice generic nav thing. At the very least, if we're going to copy something, please leave a comment explaining where it was copied from.